.gitignore

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
to strip possible system artifacts and make the repo cleaner?will delete in next commit